x-pack/plugins/fleet/public/applications/integrations/hooks/use_package_install.tsx

```diff
@@ -159,13 +186,14 @@ function usePackageInstall({
         }
       }
     },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
The only gnarly bit about the new code layout is that there is a cyclic dependency between `optionallyForceInstall` and `installPackage`, but omitting from the deps array works

Reply:
@hop-dev what about putting `optionallyForceInstall` the `installPackage` function body, as I saw this is just here to make the code more readable right?